### ISSUE TITLE
Large number of 404 errors causing delay on page load. Assistance/Insight Requested.

### ISSUE THREAD

**Issue by Clintillius** [jellyfin/jellyfin#6562]
NOTES: Windows 10 Jellyfin Server running behind a nginx reverse proxy + SSL cert using standard nginx config provided by documentation. The jellyfin server operates as expected and is 100% functional.

ISSUE: I have noticed there is a delay during my page load which occurs on every device. When I inspect the network and traffic, I see hundreds of 404 failed requests, nearly all with filenames of "auto(randomString).woff2" or "auto(randomString).bundle.js". These errors only appear when loading the webpage initially. Traversing from there does not result in any further errors. Here is what I see.

![4ooy8heug4o71](https://user-images.githubusercontent.com/29452502/133866219-63c7e70d-6a82-42cb-a4f8-536be3096c6b.png)

The delay caused ranges from about 1 second to about 4-5 seconds. Anyone have insight on what these might be and how they can be removed/disabled/not requested during page load?

**Comment by Clintillius** [comment]
Here is a followup from what I have found out.

All the 404 errors are produced by a single file. "serviceworker.js" I have reviewed this file, but with my lack of code understanding and not a single comment included has left me without an answer.

However, with my caveman testing approach, I simply renamed the file to "serviceworker.old" to force it to fail loading, and now that delay is gone. The issue is this file, and even with it no longer loading in, everything is working on every device, which I can only assume means it's not important??

If anyone has any insight on this file, I would appreciate it. The problem seems to be that it contains static links to elements I don't have. The same hundreds of requests that I see 404.
Here is a snippet:

```js
}), !1), x = [{
'revision': 'b82f18a765bd651354e61c11b701cbbe',
'url': 'auto0095962de230e8e76d21c78c2130ce89.woff2'
}, {
'revision': '2328f2e4dd46f3e73c1580d17f150e5d',
'url': 'auto00cd17f35016e45af8d4997c1a37d95c.woff2'
}, {
'revision': 'd469070b36ad4f6ff06796331d864de2',
'url': 'auto013d88d33df6093c608e5fc9c6de13ef.woff2'
}, {
'revision': 'aec5a003252f616faad8487f319784a1',
'url': 'auto014f4284992c2fee595f88a8dbb6374a.woff2'
}, {
'revision': '8a6cfeea7685cde29d34fc6eca52c7db',
'url': 'auto0176d06db401a160046e011e07e511bb.woff2'
}, {
'revision': '4a8f96d4d6f5132b10738c8bcdefb86d',
'url': 'auto01a9366fe00110272cd4a57fb1eedeac.woff2'
}, {
'revision': '49b41871da6d8b0808c5856e762ccc5f',
'url': 'auto01bd99619f5a506500ba200ece4c7065.woff2'
}, {
'revision': '365e4f1b4ab615e6beb98c3b9e708fde',
'url': 'auto01ed3d1e058adc2ab1c59262e5393d17.woff2'
}, {
'revision': '3557ea3c2c541ce3e4a58f5ff78e301b',
'url': 'auto02071f01b76e11e0289b5e9c5482629f.woff2'
```

**Comment by Clintillius** [comment]
I see now that this was a known bug that I stumbled on and was digging into. This issue has been resolved in version 10.7.7

"jellyfin/jellyfin-web#2759 [@thornbill] Fix serviceworker paths"

Thank you JF team for your hard work on this project! This issue has been closed.
